docs/source/dataset_streaming.rst

```diff
@@ -93,6 +96,29 @@ Tokenizers are written in Rust and use parallelism to speed up tokenization. To
     {'input_ids': [101, 11047, 10497, 7869, 2352...], 'token_type_ids': [0, 0, 0, 0, 0...], 'attention_mask': [1, 1, 1, 1, 1...]}
 
 
+Split your dataset with ``take`` and ``skip``
+--------------------------------------------------
+
+You can split your dataset by taking or skipping the first ``n`` examples.
+
+You can create new dataset with the first ``n`` examples using :func:`datasets.IterableDataset.take`, or you can get a dataset with the rest of the examples by skipping the first ``n`` examples with :func:`datasets.IterableDataset.skip`:
+
+
+.. code-block::
+
+    >>> dataset_head = dataset.take(2)
+    >>> list(dataset_head)
+    [{'id': 0, 'text': 'Mtendere Village was...'}, '{id': 1, 'text': 'Lily James cannot fight the music...'}]
+    >>> # You can also create splits from a shuffled dataset
+    >>> train_dataset = shuffled_dataset.skip(1000)
+    >>> eval_dataset = shuffled_dataset.take(1000)
+
+Some things to keep in mind:
+
+- When you use ``skip``, on a dataset, then iterating on the new dataset will take some time to start. This is because under the hood it has to iterate over the skipped examples first.
+- Using ``take`` (or ``skip``) prevents future calls to ``shuffle`` from shuffling the dataset shards order, otherwise the taken examples could come from other shards. In this case it only uses the shuffle buffer. Therefore it is advised to shuffle the dataset before splitting using ``take`` or ``skip``.
```

Comment:
is it possible to link to the location in the docs where "shuffle buffer" is defined? since it's a bit of a new concept (to me at least), we might want to help the user :)